lua/core/project.lua

```diff
@@ -0,0 +1,49 @@
+local M = {}
+function M.config()
+  lvim.builtin.project = {
+    --- This is on by default since it's currently the expected behavior.
+    ---@usage set to false to disable project.nvim.
+    active = true,
+
+    -- Manual mode doesn't automatically change your root directory, so you have
+    -- the option to manually do so using `:ProjectRoot` command.
+    manual_mode = false,
+
+    -- Methods of detecting the root directory. **"lsp"** uses the native neovim
+    -- lsp, while **"pattern"** uses vim-rooter like glob pattern matching. Here
+    -- order matters: if one is not detected, the other is used as fallback. You
+    -- can also delete or rearangne the detection methods.
+    detection_methods = { "lsp", "pattern" },
+
+    -- All the patterns used to detect root dir, when **"pattern"** is in
+    -- detection_methods
+    patterns = { ".git", "_darcs", ".hg", ".bzr", ".svn", "Makefile", "package.json" },
+
+    -- When set to false, you will get a message when project.nvim changes your
+    -- directory.
+    silent_chdir = true,
+  }
+end
+function M.setup()
+  require("project_nvim").setup {
```

Comment:
Also, instead of having to match manually all keys, it would be better if you put the plugin's setup config in a separate table, which you would be able to pass directly here instead.

Reply:
Done